packages/nextjs/package.json

```diff
@@ -11,28 +11,48 @@
   },
   "main": "build/cjs/index.server.js",
   "module": "build/esm/index.server.js",
-  "browser": "build/esm/index.client.js",
   "types": "build/types/index.types.d.ts",
   "exports": {
     "./package.json": "./package.json",
     ".": {
+      "edge": {
+        "import": "./build/esm/edge/index.js",
+        "require": "./build/cjs/edge/index.js",
+        "default": "./build/esm/edge/index.js"
+      },
+      "edge-light": {
+        "import": "./build/esm/edge/index.js",
+        "require": "./build/cjs/edge/index.js",
+        "default": "./build/esm/edge/index.js"
+      },
+      "worker": {
+        "import": "./build/esm/edge/index.js",
+        "require": "./build/cjs/edge/index.js",
+        "default": "./build/esm/edge/index.js"
+      },
+      "workerd": {
+        "import": "./build/esm/edge/index.js",
+        "require": "./build/cjs/edge/index.js",
+        "default": "./build/esm/edge/index.js"
+      },
       "browser": {
         "import": "./build/esm/index.client.js",
         "require": "./build/cjs/index.client.js"
       },
       "node": "./build/cjs/index.server.js",
+      "import": "./build/esm/index.server.js",
```

Comment:
Hi @lforst
I noticed in this PR that `node` exports take precedence over `import`. Is this expected? This causes the resolution to always fall back to the CommonJS version in a Node environment, regardless of whether ESM is used.